RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -54,36 +57,51 @@ struct PurchaseInformation {
     /// - `false` for purchases outside the trial period.
     let isTrial: Bool
 
-    /// Indicates whether the purchased subscription is cancelled
+    /// Indicates wheter the purchased subscripcion is cancelled
```

Comment:
/// Indicates whether the purchased subscription is cancelled

Reply:
I was missing an update in the branch